flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -277,6 +277,29 @@ public StreamExecutionEnvironment enableCheckpointing(long interval) {
 	/**
 	 * Enables checkpointing for the streaming job. The distributed state of the streaming
 	 * dataflow will be periodically snapshotted. In case of a failure, the streaming
+	 * dataflow will be restarted from the latest completed checkpoint. This method selects
```

Comment:
I think its sufficient to have the configuration parameter in the `CheckpointConfig`, without any convenience methods in the  `StreamExecutionEnvironment`. 
The `StreamExecutionEnvironment` is getting hard to maintain with those changes.